src/OrganisationRegistry.UI/app/organisation/management/overview/overview.template.html

```diff
@@ -0,0 +1,43 @@
+<div class="label-card__content-wrapper">
+  <div class="cta-title">
+      <h3 class="h3 cta-title__title">Stopzetting in de KBO</h3>
+  </div>
+  <div class="properties ww-info">
+    <div class="properties__column">
+      <dl *ngIf="organisationTermination.date">
+        <dt class="properties__label">Status:</dt>
+        <dd id="status" class="properties__data">De organisatie is stopgezet in de KBO.</dd>
+
+        <dt class="properties__label">Datum:</dt>
+        <dd class="properties__data" *ngIf="organisationTermination.date">{{organisationTermination.date}}</dd>
+
+        <dt class="properties__label">Code:</dt>
+        <dd class="properties__data" *ngIf="organisationTermination.code">{{organisationTermination.code}}</dd>
+
+        <dt class="properties__label">Reden:</dt>
+        <dd class="properties__data" *ngIf="organisationTermination.reason">{{organisationTermination.reason}}</dd>
+
+      </dl>
+      <dl *ngIf="!organisationTermination.date">
+        <dt class="properties__label">Status:</dt>
+        <dd id="status" class="properties__data" *ngIf="!organisation.kboNumber">De organisatie is niet gekoppeld aan een KBO nummer.</dd>
+        <dd id="status" class="properties__data" *ngIf="organisation.kboNumber">De organisatie is niet stopgezet in de KBO.</dd>
+      </dl>
+    </div>
+  </div>
+
+  <button class="button form__button"
+          type="button"
+          [disabled]="!syncEnabled"
+          [ngClass]="{'button--disabled': !syncEnabled, 'button--loading': isLoading}"
+          (click)="sync()">
+    <span>Synchroniseren met KBO</span>
```

Comment:
sometimes we write `de KBO` and sometimes just `KBO`, which one should we pick to make everything consistent with?

Reply:
Full-length sentences vs short command descriptions. Not sure if it makes sense to pick one style for both info text and button text. We can make it consistent among a certain type, if that's not already the case.